pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.black]
+# 88 is the default, doesn't hurt to set it explicitly
+line-length = 88
```

Comment:
How about 100? :) 88 seems to be extremely strick :)

Reply:
Done. Personally I find it easier to work with shorter lines. [Explanation for the black's default limit.](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length)